backpack/core/derivatives/lstm.py

```diff
@@ -0,0 +1,358 @@
+"""Partial derivatives for nn.LSTM."""
+from typing import Tuple
+
+from torch import Tensor, allclose, cat, einsum, sigmoid, tanh, zeros
+from torch.nn import LSTM
+
+from backpack.core.derivatives.basederivatives import BaseParameterDerivatives
+
+
+class LSTMDerivatives(BaseParameterDerivatives):
+    """Partial derivatives for nn.LSTM layer.
+
+    Index conventions:
+    ------------------
+    * t: Sequence dimension
+    * v: Free dimension
+    * n: Batch dimension
+    * h: Output dimension
+    * i: Input dimension
+    """
+
+    @staticmethod
+    def _check_parameters(module: LSTM) -> None:
+        """Check the parameters of module.
+
+        Args:
+            module: module which to check
+
+        Raises:
+            NotImplementedError: If any parameter of module does not match expectation
+        """
+        if module.num_layers > 1:
+            raise NotImplementedError("only num_layers = 1 is supported")
+        if module.bias is not True:
+            raise NotImplementedError("only bias = True is supported")
+        if module.batch_first is not False:
+            raise NotImplementedError("only batch_first = False is supported")
+        if not module.dropout == 0:
+            raise NotImplementedError("only dropout = 0 is supported")
+        if module.bidirectional is not False:
+            raise NotImplementedError("only bidirectional = False is supported")
+        if not module.proj_size == 0:
+            raise NotImplementedError("only proj_size = 0 is supported")
+
+    @staticmethod
+    def _forward_pass(
+        module: LSTM, mat: Tensor
+    ) -> Tuple[Tensor, Tensor, Tensor, Tensor]:
```

Comment:
* Could you explain the necessity to re-evaluate (parts of) the forward pass in the docstring 
* Regarding naming: Is `ifgo` a meaningful name for someone that knows more than me about LSTMs?

Reply:
I wrote down variables and the forward pass in class docstring.
I added a docstring to `_forward_pass()`.

`ifgo` is just a concatenation of the hidden variables i, f, g, o.
Please check whether that is clear from the docstrings. If not I need to change it.